content/3_Silver/Prefix_Sums.mdx

```diff
@@ -92,29 +101,37 @@ export const problems = {
 			['Prefix Sums'],
 			'usaco-104'
 		),
-	],
-	related: [
 		new Problem(
-			'Silver',
-			'My Cow Ate My Homework',
-			'762',
-			'Easy',
+			'KS@Google KickStart',
+			'Candies (Test Set 1)',
+			'https://codingcompetitions.withgoogle.com/kickstart/round/000000000019ff43/0000000000337b4d',
+			'Hard',
 			false,
```

Comment:
I think this trick/modification to psums is quite useful. If we remove from module should we star the problem? (or is there another starred problem that introduces this trick?)

Reply:
wait, were you referring to "Candies" or "My Cow Ate my Homework"?

Reply:
Candies (or wherever the `a[i]*i` psum array explanation is)